libraries-security/src/main/java/com/baeldung/digitalsignature/Utils.java

```diff
@@ -0,0 +1,33 @@
+package com.baeldung.digitalsignature;
+
+import java.io.FileInputStream;
+import java.security.KeyStore;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.cert.Certificate;
+
+public class Utils {
+
+    private static final String STORE_TYPE = "PKCS12";
+    private static final char[] PASSWORD = "changeit".toCharArray();
+    private static final String SENDER_KEYSTORE = "sender_keystore.p12";
+    private static final String SENDER_ALIAS = "senderKeyPair";
+
+    public static final String signingAlgorithm = "SHA256withRSA";
```

Comment:
if this is `static final` then should it be renamed `SIGNING_ALGORITHM`? I take it that there's no existing file with that constant already in it in the Java library?